app/client/src/components/pages/Community/components/tabs/AquaticLife.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+// components
+import TabErrorBoundary from 'components/shared/ErrorBoundary/TabErrorBoundary';
+import AssessmentSummary from 'components/shared/AssessmentSummary';
+import WaterbodyList from 'components/shared/WaterbodyList';
+// contexts
+import { LocationSearchContext } from 'contexts/locationSearch';
+// utilities
+import { useWaterbodyFeatures, useWaterbodyOnMap } from 'utils/hooks';
+
+// --- styled components ---
+const Container = styled.div`
+  padding: 1em;
```

Comment:
I'd add the same top-padding you have on the `Eating Fish` component, so there's not extra space at the top when the "Show Text" toggle is switched off:

```suggestion
  padding: 0 1em 1em;
```